### ISSUE TITLE
Wrong boolean identifier (ASN1 Encoding) ?

### ISSUE THREAD

**Issue by fferrere** [fjames86/cerberus#3]
Hello,

The function `encode-boolean` (asn1.lisp file) write 0 as boolean identifier : 
```
(defun encode-boolean (stream value)
  (encode-identifier stream 0)
  (encode-length stream 1)
  (write-byte (if value #xff 0) stream))

```
But, according to the ITU-T recommendations (page 6), the type (identifier) for boolean data is 1.

[ITU Document](https://www.itu.int/rec/dologin_pub.asp?lang=e&id=T-REC-X.690-200811-S!!PDF-E&type=items).

I'm right ?

Thanks

**Comment by fjames86** [comment]
Good spot, yes I believe you are correct and this is a bug. Are you seeing a real bug caused by this incorrect behaviour? Last time I used the library (some years ago) it worked fine. I don't know if booleans are used anywhere by the client (maybe they aren't - I haven't checked) or perhaps the servers I have used previously were more forgiving. 

Can you prepare a PR to fix this please?

**Comment by fferrere** [comment]
No, i haven't see any bugs, cause the client don't use booleans.
But i need booleans to implement PAC-REQUEST preauth (Microsoft MS-KILE extension).

And finally RFC-3244 (Microsoft change-password extension).

I won't be able to prepare a PR until next week (vacation).
